test/integration/targets/docker_image/tasks/tests/options.yml

```diff
@@ -0,0 +1,183 @@
+---
+- name: Registering image name
+  set_fact:
+    iname: "{{ name_prefix ~ '-options' }}"
+    iname_1: "{{ name_prefix ~ '-options-1' }}"
+
+- name: Registering image name
+  set_fact:
+    inames: "{{ inames }} + [iname, iname_1]"
+
+####################################################################
+## buildargs #######################################################
+####################################################################
+
+- name: buildargs
+  docker_image:
+    path: "{{ role_path }}/files"
+    name: "{{ iname }}"
+    buildargs:
+      TEST1: val1
+      TEST2: val2
+      TEST3: "True"
+  register: buildargs_1
+
+- name: buildargs (idempotency)
+  docker_image:
+    path: "{{ role_path }}/files"
+    name: "{{ iname }}"
+    buildargs:
+      TEST1: val1
+      TEST2: val2
+      TEST3: "True"
+  register: buildargs_2
+
+- name: cleanup
+  docker_image:
+    name: "{{ iname }}"
+    state: absent
+
+- assert:
+    that:
+    - buildargs_1 is changed
+    - buildargs_2 is not changed
+  when: docker_py_version is version('1.6.0', '>=')
+
+- assert:
+    that:
+    - buildargs_1 is failed
+    - buildargs_2 is failed
+  when: docker_py_version is version('1.6.0', '<')
+
+####################################################################
+## container-limits ################################################
```

Comment:
container-limits → container_limits (here and below)

Reply:
Done.